src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -282,7 +282,7 @@ class InputsEmbedder::IInputsEmbedder {
                     reshaped_image.set_shape({1, image_shape.at(0), image_shape.at(1), image_shape.at(2)});
                     break;
                 case 4: break;
-                default: OPENVINO_THROW("Input image must have [NHWC] or [HWC] layout");
+                default: OPENVINO_THROW("Input image must have [NHWC] or [HWC] layout, given image rank is ", image_shape.size());
```

Comment:
```suggestion
                default: OPENVINO_THROW("Input image must have [NHWC] or [HWC] layout, given image shape is ", image_shape);
```
It takes some effort to realize that the reader i supposed to match the reported number with the number of letters in `[]`. At the same time when the shape is printed it's already is formatted with `[]` plus giving specific shape simplifies identification of the image if there were many of them.

Reply:
agree, forgot that image shape can also be easily printed (let's check)